exercises/06_count_words/06_count_words_test.go

```diff
@@ -0,0 +1,23 @@
+package main
+
+import "testing"
+
+func TestCountWords(t *testing.T) {
+	for _, tc := range []struct {
+		name string
+		str  string
+		want int
+	}{
+		{"empty", "", 0},
+		{"Hello, world", "hello, world", 2},
+		{"Thats a lot of text", `Java is a language that basically outsourced memory management entirely to its garbage collector. This turned out to be a big mistake. However, to be able to explain why, I need to cover more of the details.
```

Comment:
Why do you need a test with a long of text? Why smaller test cases wouldn't work?